src/frequenz/client/common/metric/__init__.py

```diff
@@ -13,7 +13,18 @@
 
 
 class Metric(Enum):
-    """List of supported metrics."""
+    """List of supported metrics.
+
+    AC energy metrics information:
+    * This energy metric is reported directly from the component, and not a
+    result of aggregations in our systems. If a component does not have this
+    metric, this field cannot be populated.
+    * Components that provide energy metrics reset this metric from time to
+    time. This behaviour is specific to each component model. E.g., some
+    components reset it on UTC 00:00:00.
+    * This energy metric does not specify the timestamp since when the energy
+    was being accumulated, and therefore can be inconsistent.
```

Comment:
Feels almost like "USE AT YOUR OWN RISK" disclaimer XD

Reply:
In a sense, yes. We decided to be transparent to the client, and expose whatever metric the components provide. We can sanitise a few, but not all, e.g., energy.

I guess we will get more user feedback for this field in the future ..